### PR TITLE
Introduced Abandoned status in EIP-1

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: _data/statuses.yaml [hunk 1]
```diff
@@ -3,5 +3,7 @@
 - Accepted
 - Final
 - Active
+- Abandoned
 - Deferred
+- Rejected
```
Comment (axic) [inline _data/statuses.yaml]: Also fixes this missing status.

**Comment by MadeofTin** [comment]
It may be nice if after a certain time of inactivity (or just set time the status hasn't changed) this would automatically be applied, so the effort is required to resurrect rather than requiring an active hand to apply the abandoned label.

**Comment by cdetrio** [comment]
There used to be a `Withdrawn` status which was basically the same, but it was removed in https://github.com/ethereum/EIPs/pull/1100

**Comment by axic** [comment]
Oh, I think Withdrawn or Abandoned makes sense.

**Comment by axic** [comment]
@nicksavers @Arachnid any comments?

**Comment by tjayrush** [comment]
I think this is a great idea. It's remarkably difficult, as a semi-outsider to the process who visits periodically, to have any idea of what's actually important. An automatically assigned 'Abandoned' (assigned after X weeks/months/years) would be helpful. I also think EIP authors should have the ability to assign 'Withdrawn'. The two things are different. One signifies lack of interest from the community. The other signifies a change of mind by the author.

**Comment by MadeofTin** [comment]
@nicksavers left a relevant comment on a separate thread.  https://github.com/ethereum/EIPs/pull/1991#discussion_r289431874 

Perhaps there is a case to add both?
 * Inactive/Abandoned
   * If an EIP is not updated by the author for 6 months, it is the editors prerogative to move the proposal to 'inactive'.
 * Withdrawn
   * For when an author actively withdraws the requested EIP due to a changing their mind or in response to community feedback.

A practical example of when I would use Withdrawn over Inactive is if my proposal https://github.com/ethereum/EIPs/pull/2025 doesn't receive enough support from the community. I will mark it as withdrawn. It is a good option to have imo, even if it has been underutilized in the past.

**Comment by nicksavers** [comment]
Yeah I'd like this idea. But there would need to be an expiration date required and this needs to be updated manually by the author as long as it is in Draft. If the 6 months pass, an editor or anybody (with help from the automerger) could set the status to Abandoned. This might help to clean up a lot of Drafts.

Not a fan of Withdrawn as it's not that different from being Abandoned where the author would just outright state that they are abandoning it.

**Comment by axic** [comment]
@nicksavers @Arachnid can we merge it now or do we need to change anything?

**Comment by Arachnid** [comment]
Can you add 'abandoned' as a state transition in the section describing those?

**Comment by axic** [comment]
> Can you add 'abandoned' as a state transition in the section describing those?

Not sure how, it is listed as an "exceptional status". The section is not clear in general, but my interpretation is that it is possible to move from multiple status into exceptional ones.

In the case of Abandoned, I'd say it is possible to move there from Draft, Last Call, Deferred and Accepted. But not possible to move from Final, Active, Rejected, Superseded.

**Comment by xinbenlv** [comment]
Agree with the proposer that a status will be good. Now that the PR is merged, can we further discuss what causes an "abandoned" status? For example, we can require the EIP author to update the text itself, with a new "updated" date? Or will the active discussion on an issue counts as activity?

I also support that in the future if some author comes up with a similar/related idea and if they so decided and they can start to revisit / revive the abandoned draft.

**Pull request merged**